src/Language/Bitcoin/Script/Descriptors/Utils.hs

```diff
@@ -196,6 +244,17 @@ outputDescriptorScripts =
                         , txWitnessScript = Just theScript
                         }
         Combo _kd -> Nothing
+        P2TR kd td -> do
+            taprootOutput <- taprootDescriptorOutput kd td
+            let theScriptPubKey =
+                    encodeOutput $
+                        taprootScriptOutput taprootOutput
+            pure
+                TransactionScripts
+                    { txScriptPubKey = theScriptPubKey
+                    , txRedeemScript = Nothing
+                    , txWitnessScript = Nothing
```

Comment:
IMO we should not support `P2TR` with this function.  Based on the spec there is no way to construct `txWitnessScript` just using the descriptor (we need a `TaprootWitness`).

Reply:
Thanks, I've changed it to now fail on P2TR.